lab-kyle/test/hello-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const hello = require(__dirname + '/../lib/greet');
```

Comment:
Its also important to keep your variable names consistant when working with a module in different files in a project